assets/js/modules/analytics/datastore/accounts.js

```diff
@@ -0,0 +1,178 @@
+/**
+ * modules/analytics data store: accounts.
+ *
+ * Site Kit by Google, Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * External dependencies
+ */
+import invariant from 'invariant';
+
+/**
+ * Internal dependencies
+ */
+import API from 'googlesitekit-api';
+
+// Actions
+const FETCH_ACCOUNTS_PROPERTIES_PROFILES = 'FETCH_ACCOUNTS_PROPERTIES_PROFILES';
+const RECEIVE_ACCOUNTS_PROPERTIES_PROFILES = 'RECEIVE_ACCOUNTS_PROPERTIES_PROFILES';
+const RECEIVE_ACCOUNTS_PROPERTIES_PROFILES_FAILED = 'RECEIVE_ACCOUNTS_PROPERTIES_PROFILES_FAILED';
+
+export const INITIAL_STATE = {
+	accounts: undefined,
+	isFetchingAccountsPropertiesProfiles: false,
+};
+
+export const actions = {
+	*fetchAccountsPropertiesProfiles() {
+		return {
+			payload: {},
+			type: FETCH_ACCOUNTS_PROPERTIES_PROFILES,
+		};
+	},
+
+	receiveAccountsPropertiesProfiles( { accounts, properties, profiles } ) {
+		return {
+			payload: { accounts, properties, profiles },
+			type: RECEIVE_ACCOUNTS_PROPERTIES_PROFILES,
+		};
+	},
+
+	receiveAccountsPropertiesProfilesFailed( error ) {
+		invariant( error, 'error is required.' );
+
+		return {
+			payload: { error },
+			type: RECEIVE_ACCOUNTS_PROPERTIES_PROFILES_FAILED,
+		};
+	},
+};
+
+export const controls = {
+	[ FETCH_ACCOUNTS_PROPERTIES_PROFILES ]: () => {
+		return API.get( 'modules', 'analytics', 'accounts-properties-profiles' );
+	},
+};
+
+export const reducer = ( state, action ) => {
+	switch ( action.type ) {
+		case FETCH_ACCOUNTS_PROPERTIES_PROFILES: {
+			return {
+				...state,
+				isFetchingAccountsPropertiesProfiles: true,
+			};
+		}
+
+		case RECEIVE_ACCOUNTS_PROPERTIES_PROFILES: {
+			const { accounts, properties, profiles } = action.payload;
+
+			return {
+				...state,
+				accounts,
+				isFetchingAccountsPropertiesProfiles: false,
+				properties,
+				profiles,
+			};
+		}
+
+		case RECEIVE_ACCOUNTS_PROPERTIES_PROFILES_FAILED: {
+			const { error } = action.payload;
+
+			return {
+				...state,
+				error,
+				isFetchingAccountsPropertiesProfiles: false,
+			};
+		}
+
+		default: {
+			return { ...state };
+		}
+	}
+};
+
+export const resolvers = {
+	*getAccounts() {
+		try {
+			const response = yield actions.fetchAccountsPropertiesProfiles();
+			const { accounts, properties, profiles } = response;
+
+			yield actions.receiveAccountsPropertiesProfiles( { accounts, properties, profiles } );
```

Comment:
See above, an eventual `matchedProperty` response property should be supported here.

Reply:
Should this perhaps be a separate action? `receiveMatchedProperty` ?